magento/magento2
# Correct signature of the $customerModel

### Description (*)

New import was added to change the signature of the Customer's Model.
Previously it was signed as a ResourseModel and IDE reported that required parameters of `save()` method were missed.
Imports were reordered to pass the code style and code standards compliance tests.

### Fixed Issues (if relevant)

1. No.

### Manual testing scenarios (*)

1. Ctrl + Shift +N: Magento/Customer/Model/ResourceModel/CustomerRepository.php
2. Ctrl + G: 240
3. Verify that inspections on method `save()` https://github.com/magento/magento2/blob/35b37c04552c750bcc1016bc6972d195acf043d8/app/code/Magento/Customer/Model/ResourceModel/CustomerRepository.php#L240
do not report the problem with the required parameters.

### Questions or comments


### Contribution checklist (*)
